liblangutil/ParserBase.cpp

```diff
@@ -107,3 +166,8 @@ void ParserBase::fatalParserError(string const& _description)
 {
 	m_errorReporter.fatalParserError(SourceLocation{position(), endPosition(), source()}, _description);
```

Comment:
Please call the other function here.

Reply:
I don't understand: which other function.

Reply:
```fatalParserError(SourceLocation{position(), endPosition(), source()}, _description);```